packages/@react-types/tabs/src/index.d.ts

```diff
@@ -48,7 +48,12 @@ interface AriaTabListBase {
   isDisabled?: boolean
 }
 
-export interface AriaTabListProps<T> extends TabListProps<T>, AriaTabListBase, DOMProps, AriaLabelingProps {}
+export interface AriaTabListProps<T> extends TabListProps<T>, AriaTabListBase, DOMProps, AriaLabelingProps {
+  /**
+   * Whether the TabList is contained in a virtual scroller.
+   */
+  isVirtualized?: boolean
```

Comment:
Can we remove this option for now?

Reply:
AHH, my bad forgot to remove it from the types